src/js/filter.ts

```diff
@@ -110,6 +110,19 @@ const setUpAllFilters = (
   document
     .querySelectorAll("input[type=checkbox]")
     .forEach((option: HTMLInputElement) => {
+      if (option.id == "no-requirements-toggle") {
```

Comment:
I think this code would be better in a file `alert.ts` and specifically use a query selector for `no-requirements-toggle`. It's not a big deal that we will select the element twice. 

`filter.ts` is already complex enough, and it's useful to not make it more complex.